x/mongo/driver/topology/fsm.go

```diff
@@ -16,8 +16,12 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/description"
 )
 
-var supportedWireVersions = description.NewVersionRange(2, 9)
-var minSupportedMongoDBVersion = "2.6"
+var (
```

Comment:
Should these be const?

Reply:
`SupportedWireVersions` cannot be because it's a struct but I changed `MinSupportedMongoDBVersion`.